tests/plugins/preview/manual/previewcdn.html

```diff
@@ -0,0 +1,72 @@
+<div id="editor"></div>
+<div id="loading_box" style="margin: 20px 0"></div>
+<div id="editor_cdn">
+	<h1>Sample <span style="color: red;">document</span></h1>
+	<p>Lorem ipsum</p>
+	<div style="page-break-after:always">
+		<span style="display:none">&nbsp;</span>
+	</div>
+	<p>dolor sit amet.</p>
+</div>
+
+<script>
+	// 1. Load local editor by default with bender.
+	// 2. Remove existing instance and add script with editor from CDN.
```

Comment:
Are these steps necessary?

Reply:
Yes, because bender loads all scripts by default every time, so I delete all files and namespace so that the new instance doesn't accidentally use local files. Additionally, we are sure that the new editor instance is loaded from CDN.